Source/WebInspectorUI/UserInterface/Views/DOMTreeContentView.js

```diff
@@ -58,6 +58,12 @@ WI.DOMTreeContentView = class DOMTreeContentView extends WI.ContentView
         this._forceAppearanceButtonNavigationItem.visibilityPriority = WI.NavigationItem.VisibilityPriority.Low;
         this._forceAppearanceButtonNavigationItem.enabled = WI.cssManager.canForceAppearance();
 
+        // COMPATIBILITY (macOS 13.0, iOS 16.0): `Page.overrideUserPreference` did not exist yet.
+        this._overrideUserPreferencesNavigationItem = new WI.ActivateButtonNavigationItem("toggle-user-preferences", WI.UIString("Override user preferences"), WI.UIString("User preferences overriden"), "Images/AppearanceOverride.svg", 16, 16);
```

Comment:
I disagree with adding these setting to the "Device Settings" panel. There is a key distinction between those settings and these setting in that the settings Razvan is adding our valid web-exposed states that the user controls, where the controls in the "Device Settings" panel are, generally (yes JS is the exception in Safari via the Security settings panel), a state that users will never be in.

I understand the desire to future-proof the position of these toggles in that way, but even `prefers-reduced-data` is arguably still about CSS first, and networking second, because that query does not itself change anything about the network requests, its still up to CSS to define different content to be loaded. Ultimately these are settings (and potential future settings) that change what CSS rules are applied on a page.

Also, the device settings coming to a local session is still being massaged after some internal debate, so isn't presently landed/available anyways.